docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -1,55 +1,75 @@
-Respondent Management to Electronic Questionnaire
--------------------------------------------------
+Census Respondent Home to Census Electronic Questionnaire
+---------------------------------------------------------
 
 When a respondent is ready to take a survey hosted on the eq system, a set of details
 need to be passed to eq to setup the survey correctly. This data is wrapped inside a json web
-token which is attached to the end of a url, digitally signed and authorised by a respondent
-management system. This creates a clean interface for any respondent management system
+token which is attached to the end of a url, digitally signed and authorised by an appropriate
+client application. This creates a clean interface for any respondent management system
 to integrate with the eq system.
 
 Schema Definition
 =================
 
-Required Fields
----------------
+Census Required Runner Fields
+------------------------------
 
 The following metadata keys are always required for the survey runner, they do not appear in individual survey metadata definitions.
 
-``tx_id``
-  see :doc:`jwt_profile`
 ``iat``
   JWT Issued At claim, see https://tools.ietf.org/html/rfc7519#section-4.1.6
 ``exp``
   JWT Expiration Time claim, see https://tools.ietf.org/html/rfc7519#section-4.1.4
-``ru_ref``
-  The responding unit reference id - with checkletter appended
-``eq_id``
-  The eQ questionnaire id
+``jti``
+   A unique identifier for the JWT
+``tx_id``
+  see :doc:`jwt_profile`
+``response_id``
+  A unique identifier for the questionnaire response
+``questionnaire_id``
+  The Census Questionnaire ID
+``case_type``
+  The type of case (HH | HI | CE | CI)
+``region_code``
+  The Region Code of the questionnaire response. Format as per ISO 3166-2 (https://en.wikipedia.org/wiki/ISO_3166-2:GB) i.e. GB-ENG | GB-WLS | GB-NIR
 ``collection_exercise_sid``
   A reference number used to represent the collection exercise inside the ONS
-``form_type``
-  The particular form_type for a responding unit
 ``ru_ref``
-  The responding unit reference id - with checkletter appended.
-``ru_name``
-  The name of the responding unit. Could be a business name or person name. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
-``trad_as``
-  Temporary until wider refactor. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
+  The reporting unit reference id (e.g. UPRN)
+``account_service_url``
+  The url of the account service (e.g. RH) used to launch the survey
+``eq_id``
+  The eQ questionnaire schema id (to be removed, hardcoded by RH as "census")
+``form_type``
+  The particular form_type for a responding unit (to be removed, hardcoded by RH as "individual_gb_eng")
```

Comment:
This should be removed. There's no reference to it in runner anymore, but it won't cause any problems if we receive it. I don't see any reason to keep it in the specification.

Reply:
I've removed these